src/throttler.decorator.ts

```diff
@@ -10,7 +10,7 @@ function setThrottlerMetadata(target: any, limit: number, ttl: number): void {
 /**
  * Adds metadata to the target which will be handled by the ThrottlerGuard to
  * handle incoming requests based on the given metadata.
- * @usage @Throttle(2, 10)
```

Comment:
Rather than remove the `@` for usage how about we change `@usage` to `@example`?